webapp/store/views.py

```diff
@@ -496,3 +496,25 @@ def entity_embedded_card(entity_name):
         "embeddable-card.html",
         **context,
     )
+
+
+@store.route('/<regex("' + DETAILS_VIEW_REGEX + '"):entity_name>/icon.png')
```

Comment:
```suggestion
@store.route('/<regex("' + DETAILS_VIEW_REGEX + '"):entity_name>/icon')
```

no need for .png if we redirect (might be svg)